### PR TITLE
refactor: static assets

### LINKED ISSUE #354: Plan/Document handling static assets in storefront
# Static Assets Documentation 
### Summary description
Currently, we've got several "static" assets being used by the storefront all being handled in slightly different ways.

  *  Source Sans Pro font being served by fonts.google.com / not configurable.
  *  Favicons being served from assets.reactioincommerce.com / configurable URL via .env
  *  Product Placeholder GIF being served locally from `./src/static/` / configurable URL via .env
  *  Shop locales are being served locally from `./src/lib/utils/locales.json` / not configurable. 

**Questions**
  1. Should all assets come from the same place? (`./src/static/...`)
  2. Should all assets have configurable URLs?
  3. Should we have docs/examples of using assets w/ CDN?

### Rationale for why this feature is necessary
At the very least document the why/how around the use of these assets or create an epic around a better implementation.

### PR DESCRIPTION
**Pull request by nnnnat**
Resolves #354
Impact: **minor**
Type: **refactor|docs|chore**

## Issue
Unclear and inconsistent handling of static assets.

## Solution
  * Move all static assets into `src/static` dir
  * Updated asset path in configs/env
  * Added docs around static assets

## Breaking changes
locales.json & favicon files are now in `src/static` dir.

## Testing
1. Make sure favicon/placeholder/locales are still loading:
    * When page loads you should always have favicons now and the network path is from "localhost/static/favicons/..."
    * View product that doesn't have an image, placeholder image should render in the grid as well as PDP.
    * Add item to cart and go checkout, during the shipping address checkout step verify the `country` and `region` option are available. 
2. Read over the new static assets docs in `./docs/`.

## Notes
I nested assets in "type" directories (favicons -> static/favicons/...), now I'm thinking it might be best to just keep everything at the root `/static/` dir.

### CODE REVIEW

File: docs/static-assets.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Static Assets
+Your storefront will probably need to include static assets such as, logo images, timezone/internationalization data, favicons, etc..
+
+## Handling static assets
+The starterkit follows the approach outlined in the [NextJS documentation](https://github.com/zeit/next.js/#static-file-serving-eg-images) keeping all static assets live in the `.src/static/` folder.
```
Comment (machikoyasuda) [inline docs/static-assets.md]: [Next.js documentation](https://github.com/zeit/next.js/#static-file-serving-eg-images)

File: docs/static-assets.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+**Static Data**
+Locales, timezones, currency codes and similar data stuctures are placed in the `/static/data` directory.
+
+**Fonts**
+Out of the box the starterkit leverages [fonts.google.com](https://fonts.google.com/) for font delivery. You could however add custom fonts to a `/static/fonts/` directory if needed.
+
+**Icons**
+We feel icons are best handled as SVGs within React components and not as a static asset. 
+See [MDI](https://github.com/TeamWertarbyte/mdi-material-ui) and [RDS](
```
Comment (machikoyasuda) [inline docs/static-assets.md]: [Reaction Design System's `svg` directory](https://github.com/reactioncommerce/reaction-component-library/tree/v0.44.0/package/src/svg)

File: docs/static-assets.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Static Assets
+Your storefront will probably need to include static assets such as, logo images, timezone/internationalization data, favicons, etc..
+
+## Handling static assets
+The starterkit follows the approach outlined in the [Next.JS documentation](https://github.com/zeit/next.js/#static-file-serving-eg-images) keeping all static assets live in the `.src/static/` folder.
```
Comment (aldeed) [inline docs/static-assets.md]: Remove the word "live" and `.src` should be `/src`

File: docs/static-assets.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+**Static Data**
+Locales, timezones, currency codes and similar data stuctures are placed in the `/static/data` directory.
+
+**Fonts**
+Out of the box the starterkit leverages [fonts.google.com](https://fonts.google.com/) for font delivery. You could however add custom fonts to a `/static/fonts/` directory if needed.
+
+**Icons**
+We feel icons are best handled as SVGs within React components and not as a static asset. 
+See [MDI](https://github.com/TeamWertarbyte/mdi-material-ui) and [Reaction Design System's svg directory](https://github.com/reactioncommerce/reaction-next-starterkit/blob/3980c71b528221b3b196d446ea24bb4bc474c74d/src/lib/apollo/initApollo.js#L66) for more info.
```
Comment (aldeed) [inline docs/static-assets.md]: I feel like the second link is not the one you intended to paste there?

**Pull request merged**